### PR TITLE
doc: Add link to argument details

### PR DESCRIPTION
**Pull request by xclaesse**

### CODE REVIEW

File: docs/refman/generatormd.py [hunk 1]
```diff
@@ -161,6 +161,9 @@ def len_stripped(s: str) -> int:
             # I know, this regex is ugly but it works.
             return len(re.sub(r'\[\[(#|@)*([^\[])', r'\2', s))
 
+        def arg_anchore(arg: ArgBase) -> str:
```
Comment (bruchar1) [inline docs/refman/generatormd.py:164]: anchore? or anchor?

Comment (xclaesse) [reply docs/refman/generatormd.py]: Typo indeed. Fixed.

**Review by bruchar1 — APPROVED** [review]
LGTM

**Comment by eli-schwartz** [comment]
Pretty neat change!

**Pull request merged**